google/cloud/aiplatform/training_jobs.py

```diff
@@ -221,6 +266,18 @@ def _run_job(
                 resource ``name``
                 is populated. The Model is always uploaded into the Project
                 and Location in which this pipeline is.
+            gcs_destination (~.io.GcsDestination):
```

Comment:
Consolidated the CustomTrainingJob's version of this function by moving the parameter here.